### DISCUSSION TITLE
执行MLC课程代码时报错

### DISCUSSION THREAD

**Discussion by CdNut** [mlc-ai/mlc-zh#183] [Q&A]
本机环境ubuntu20.04
1. 参照官网TVM从源码安装教程，编译全程无报错。执行 rt_lib = tvm.build(MyModule, target="llvm") 报错：  Check failed: (bf != nullptr) is false: target.build.llvm is not enabled。但是执行 python -c "import tvm; print(tvm.target.Target('llvm').kind.name)" 可正常输出 llvm
2. 若采用 MLC 课程代码 python3 -m  pip install mlc-ai-nightly -f https://mlc.ai/wheels，可以执行 rt_lib = tvm.build(MyModule, target="llvm")。但执行 rt_mod = tvm.build(sch.mod, target="cuda") 报错：Check failed: (bf != nullptr) is false: target.build.cuda is not enabled

**Comment by Hzfengsy** [comment]
- 源码编译TVM需要打开llvm编译选项（官网编译教程有写）
- mlc-nightly不含cuda编译

**Reply by JianDan0212** [reply]
请问执行rt_lib_after = tvm.build(sch.mod, target="llvm")  命令时，提示进程已结束,退出代码-1073740940 (0xC0000374)是什么原因呀
